docs/dom.api.md

```diff
@@ -19,52 +19,56 @@ API. You will, however, [need to also handle Backbone's jQuery integration](#bac
 Returns a new HTML DOM node instance. The resulting node can be passed into the
 other DOM functions.
 
-### `appendChildren(el, children)`
+### `getEl(selector, context)`
 
-Takes the DOM node `el` and appends the rendered `children` to the end of the
-element's contents.
+Lookup the `selector` string within the DOM node for `context`. The optional
+`context` argument will come in as a DOM Node reference to run the `selector`
+search. If `context` hasn't been set, then `getEls` should search the entire
```

Comment:
True.  There's a handful of changes to make to the docs too.  I only modified them slightly so far.